google/cloud/logging_v2/_gapic.py

```diff
@@ -121,7 +121,7 @@ def write_entries(
         logger_name=None,
         resource=None,
         labels=None,
-        partial_success=False,
```

Comment:
Based on https://cloud.google.com/logging/docs/reference/v2/rest/v2/entries/write, seems partialSuccess is optional - I guess keeping default to `false` would be more correct in this case, right?

Reply:
It's hard to say which is more correct. Based on issue https://github.com/googleapis/python-logging/issues/448, I thought you were leaning towards setting `partialSuccess` to `True` by default, since it would prevent losing entire batches of logs when one is over-size.

But there's also an argument that we should just stick to the backend defaults whenever possible (and I assume this defaults to `False`? But it's not completely clear).

I could go either way, but we should probably at least be consistent about it across languages

Reply:
> It's hard to say which is more correct. Based on issue #448, I thought you were leaning towards setting `partialSuccess` to `True` by default, since it would prevent losing entire batches of logs when one is over-size.
> 
> But there's also an argument that we should just stick to the backend defaults whenever possible (and I assume this defaults to `False`? But it's not completely clear).
> 
> I could go either way, but we should probably aim to be consistent about it across languages

Oh, perhaps I was misleading - I was meaning to expose an ability for setting `partialSuccess` in configuration as we have it in other languages.